src/pages/Lend/Filter/FilterComponents/FilterRangeSlider.vue

```diff
@@ -113,7 +122,7 @@ export default {
 			return Math.max(this.minimum, this.range.min);
 		},
 		sliderMaximum() {
-			return Math.min(this.maximum, this.range.max);
+			return this.attribute === 'partner.riskRating' ? this.maximum : Math.min(this.maximum, this.range.max);
```

Comment:
This change actually breaks the UI in situations where not all star ratings are available. Try limiting the set to 10/15 loans then drag the slider, the handle and the progress bar don't match. Can we revert?

Reply:
@ksekhri Sure thing! Reverted.